kernels/0010_Conv2D.md

```diff
@@ -0,0 +1,41 @@
+# Batched Conv2D
+
+**Link:** https://github.com/cmeraki/vit.triton/blob/main/vit/kernels/conv2d.py
+
+**Author:** Romit
```

Comment:
If you want, feel free to add your full name, with a link to your twitter or LinkedIn. Eg `[Romit last_name](x.com/blabla)`.

Same in other files